utils/utils.go

```diff
@@ -61,7 +61,7 @@ func WriteTable(writer io.Writer, csvContent string) {
 
 func TruncateString(str string, maxLength int) string {
 	strLength := len(str)
-	if strLength <= maxLength {
+	if strLength <= maxLength || maxLength <= 3 {
```

Comment:
What is this change about?

Reply:
I write the unit test,when maxLength is 0,will cause panic: 
runtime error: slice bounds out of range [:-3] [recovered]
	panic: runtime error: slice bounds out of range [:-3]
So add this condition.